### DISCUSSION TITLE
Text inside mj-raw is not visible

### DISCUSSION THREAD

**Discussion by PttRulez** [mjmlio/mjml#2670] [Q&A]
Hey guys. I have some markup inside mj-raw. Image is visible but text under it is not visible. I can see it in Inspection dev tools but it's not shown on display. If I just change mj-raw to mj-text - it shows up. What kind of magic could it be?
```
<mj-raw align="center">
                    <div class="popular-video">
                        <div class="popular-video-image"
                            style="background: url(someurl') center center  no-repeat"
                        ></div>
                        <div class="popular-video-text">
                            Video Title
                        </div>
                    </div>
                   ...
```
I can put that text in div paragraph or without tag - same result

**Comment by iRyusa** [comment]
`mj-raw` will put your html directly into the DOM, so it might do invalid HTM if you don't check what's the previous tag. And `align` is not a supported attribute on mj-raw.

You might t need mj-raw in your case just use mj-text instead.
